.github/actions/plan/modes.ts

```diff
@@ -63,10 +64,28 @@ export const code = {
 } satisfies Modes;
 
 export const build = {
-  build: {
-    name: "build",
+  buildPeer: {
+    name: "build peer",
     cargoCommand: "build",
     cargoArgs: "--locked --workspace --release",
     cargoCacheKey: "release-build",
+    artifactSelector: "peer",
+  },
+  // buildRuntime: {
+  //   name: "build runtime",
+  //   cargoCommand: "build",
+  //   cargoArgs: "--locked --workspace --release --package humanode-runtime",
+  //   cargoCacheKey: "runtime",
+  //   platformIndependent: true,
+  //   artifactSelector: "runtime",
+  // },
+  buildRuntimeEvmTracing: {
+    name: "build runtime with EVM tracing",
+    cargoCommand: "build",
+    cargoArgs:
+      "--locked --workspace --release --package humanode-runtime --features evm-tracing",
```

Comment:
Do not use `--package` if possible as it can scramble features is a way we not checking for (we are not ready to use this kind of target selection until we add CI tests for this), use the `--bin` (or similar) target instead

Reply:
I think it will be ideal for us to build with the `--lib` flag; I almost forgot it exists, I’ll add it.